src/plots/cartesian/axis_ids.js

```diff
@@ -136,3 +136,26 @@ exports.ref2id = function(ar) {
     // return the axis ID. Otherwise it returns false.
     return (/^[xyz]/.test(ar)) ? ar.split(' ')[0] : false;
 };
+
+
+exports.isLinked = function(fullLayout, axId) {
+    var linked = false;
+
+    (fullLayout._axisConstraintGroups || []).forEach(function(e) {
+        if(e[axId]) {
+            linked = true;
+            return;
```

Comment:
This `return` (and in the next loop) don't do anything... maybe make the `forEach` calls into `for` loops`?

Reply:
Good call. Revised in https://github.com/plotly/plotly.js/pull/5329/commits/2bc8420367b84d9b788ac95582650f807f1187cb.